### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by n0k0m3**
Added Prerequisites (Boost) and installation instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,15 +19,36 @@ Email:rpoteau2010@my.fit.edu
...
+cd boost_1_69_0
+sudo ./bootstrap.sh
+sudo ./b2 install
+```
 
 ### Installation
 Make sure Eigen is installed. Download or clone the repository into a 
-folder and run make command in the project's root directory.
+folder and run `make` command in the project's root directory 
+(you can also use `sudo make` if the command doesn't work).
```
Comment (jgoldfar) [inline README.md:51]: Don't get in the habit of using `sudo` to work around a build error, since permissions shouldn't be an issue... Why was this necessary?

File: README.md [hunk 2]
```diff
@@ -37,6 +58,14 @@ When you create your own examples, just drop it into the examples
 folder and run the make file in the root directory this will create an
 executable which you can run from the bin folder. 
 
+To run the executable, you need libparamid in your system's library
+```
+# At QLOPT root directory
+cd build
+sudo cp libparamid.* /usr/lib/
```
Comment (jgoldfar) [inline README.md:65]: This indicates a linking issue (AFAIK)

File: README.md (hunk 1 above)
Comment (n0k0m3) [reply README.md:51]: Installation of eigen requires writing to /usr/include, in some system this is prohibited for user account and requires su privileges

Comment (jgoldfar) [reply README.md:51]: `make` should just build the library, not install it. But you're correct that if you're admin on your machine, this is fine (unless you need multiple versions of Eigen installed) but for small libraries it's fine to "vendor" that dependency by installing it locally. This is also handy when working on shared servers. I do this in the PR #4 to avoid requiring sudo privileges, which decreases test performance.

Comment (robypoteau) [reply README.md:51]: you don't really have to install eigen. You can copy and paste it into you project. Eigen is a bunch of header files.
